app/models/giftaid/GiftAidPaymentsModel.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.giftaid
```

Comment:
<img width="493" alt="Screenshot 2021-04-09 at 11 13 55" src="https://user-images.githubusercontent.com/60218795/114165712-c05b1880-9924-11eb-8207-e4f87208e57a.png">

folder name should be giftAid